webapp/src/Entity/Testcase.php

```diff
@@ -52,7 +52,7 @@ class Testcase
      * @var int
      * @ORM\Column(type="integer", name="probid", length=4,
      *     options={"comment"="Corresponding problem ID", "unsigned"=true},
-     *     nullable=false)
+     *     nullable=true)
```

Comment:
Just to check: this doesn't break references when fetching deleted testcases for old judgings?

Reply:
You mean on any of the submissions overview page or the submission page? I have tested both and added the warning (see commit description) to the latter.